site/src/lib/blocks.ts

```diff
@@ -0,0 +1,135 @@
+/** @todo type as JSON object */
+export type BlockProps = object;
+
+/** @todo make part of blockprotocol package and publish */
+export type BlockVariant = {
+  description?: string;
+  displayName?: string;
+  icon?: string;
+  properties?: BlockProps;
+};
+
+/** @todo make part of blockprotocol package and publish */
+export type BlockMetadata = {
```

Comment:
These can all be imported from "blockprotocol" now (you'll need to add `packagePath` locally) - `blockprotocol` will need adding as a dependency

Reply:
Added here e41bce7

Reply:
Small follow up, sorry for not being clear - I think we should add `packagePath` in this file, e.g.

```typescript
type ExpandedBlockMetadata = BlockMetadata & { packagePath: string }
```

because `packagePath` is just something temporary we're putting in, and we generate it here, so we don't want to make it required for other people using the type as a guide for what to include. We should only include things in `blockprotocol` that are consistent with what the spec says.

Reply:
There are a couple of other type errors blocking merge I suggest can be fixed in the same way (adding extra fields in `site` rather than to `blockprotocol`, where they aren't something we want to mention in the spec)

Reply:
Ahh I see. Will update

Reply:
a651d1c6ef43762